docs/introduction.rst

```diff
@@ -35,6 +35,43 @@ In each recipe we indicate where to find the corresponding directories in the
 archive with square brackets after the section title (e.g. [Input:
 `recipes/basics/firstcalc/`]).
 
+
+Installing from conda-forge
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Probably needs a line of introduction for those unaware of anaconda/conda (and perhaps a link to conda.io or the wikipedia entry).